docs/CONFIG-VARS.md

```diff
@@ -367,6 +367,7 @@ The following variables are used to describe the machine targets for the SAS Viy
 | node_labels | Labels applied to nodes in your cluster | map(list(string)) | | See [Labels/Taints](#labelstaints) below for more information. |
 | node_taints | Taints applied to nodes in your cluster | map(list(string)) | | See [Labels/Taints](#labelstaints) below for more information. |
 | control_plane_ssh_key_name | Name for generated control plane SSH key | string | "cp_ssh" | |
+| containerd_version | Version of containerd to be installed  | string | "1.6.20-1" | Set as an empty string to use the latest upstream version from the [Docker repository](https://download.docker.com/linux/ubuntu/dists/) |
```

Comment:
What/Where/How do I find the version list and determine what to set this value? I went to the download link and it's just a listing of items. Need to tell the user how to find the versions and release info.

Reply:
That's probably a good idea, I can add details/notes here to help users find what containerd versions are available since there is not an easy/user friend way for users to browse the Docker apt repo.

But the summary would be:

1. Go to the link provided
2. Select your Ubuntu dist from the list Jammy (22) or Focal (20)
3. The navigate to stable>binary-amd64>Packages
4. Packages is a text file where you can look at all the containerd versions that are available.

It may be simpler to provide two links one for Jammy and another for Focal so that the user just has to click "Packages" so we can leave out the verbose navigation steps